rel/overlay/bin/remsh

```diff
@@ -49,10 +49,10 @@ NODE="${NODE:-$DEFAULT_NODE}"
 
 # If present, extract cookie from ERL_FLAGS
 # This is used by the CouchDB Dockerfile and Helm chart
-COOKIE=$(echo "$ERL_FLAGS" | sed 's/^.*setcookie \([^ ][^ ]*\).*$/\1/g')
+COOKIE=$(echo "$ERL_FLAGS" | sed 's/^.*setcookie[ ]*\(['\''"]\)*\([^'\''"]*\)\1*.*$/\2/')
```

Comment:
I was testing this before the code was updated.

```
#!/bin/bash
COOKIE=$(echo "$ERL_FLAGS" | sed -E '
s/.*-setcookie[ ]*["]([^"]*)["].*/\1/;
s/.*-setcookie[ ]*([^ ]*).*/\1/;
')

echo "COOKIE='${COOKIE}'"

ERL_FLAGS_CLEAN=$(echo "$ERL_FLAGS" | sed -E '
s/-setcookie[ ]*[^"]*.*["]//
s/-setcookie[ ]*[^ ]*//
s/-name[ ]*[^ ]*//
')

echo "ERL_FLAGS_CLEAN='${ERL_FLAGS_CLEAN}'"
```

```
bash-3.2$ ERL_FLAGS='ddd -setcookie "secret  123" -name node1@mydomain.com hello' ./test
COOKIE='secret  123'
ERL_FLAGS_CLEAN='ddd secret  123"  hello'
bash-3.2$ micro test
bash-3.2$ ERL_FLAGS='ddd -setcookie "secret  123" -name node1@mydomain.com hello' ./test
COOKIE='secret  123'
ERL_FLAGS_CLEAN='ddd secret  123"  hello'
bash-3.2$ ERL_FLAGS='ddd -setcookie "secret  123" -name node1@mydomain.com hello' ./test
COOKIE='secret  123'
ERL_FLAGS_CLEAN='ddd secret  123"  hello'
bash-3.2$ micro test
bash-3.2$ ERL_FLAGS='ddd -setcookie "secret  123" -name node1@mydomain.com hello' ./test
COOKIE='secret  123'
ERL_FLAGS_CLEAN='ddd   hello'
bash-3.2$ ERL_FLAGS='ddd -setcookie "secret  123"' ./test
COOKIE='secret  123'
ERL_FLAGS_CLEAN='ddd '
bash-3.2$ ERL_FLAGS='ddd -setcookie "secret" -name node1@mydomain.com hello' ./test
COOKIE='secret'
ERL_FLAGS_CLEAN='ddd   hello'
bash-3.2$ ERL_FLAGS='ddd -setcookie "secret"' ./test
COOKIE='secret'
ERL_FLAGS_CLEAN='ddd '
bash-3.2$ ERL_FLAGS='ddd -setcookie secret -name node1@mydomain.com hello' ./test
COOKIE='secret'
ERL_FLAGS_CLEAN='ddd   hello'
bash-3.2$ ERL_FLAGS='ddd -setcookie secret' ./test
COOKIE='secret'
ERL_FLAGS_CLEAN='ddd '
```

Reply:
`sed -E` / `sed -r` is very powerful and helped me remove the if statement. Thank you!